lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -49,7 +49,7 @@ blockchain_configuration:
   linear_fees:
     constant: 42
     coefficient: 0
-    certificate: 0
+    certificate: 1
```

Comment:
This is actually interesting. We should probably use some non-null fee for the coefficient part as well.